docs/developer/style.md

```diff
@@ -104,10 +302,10 @@ for _, tt := range tests {
 </tbody>
 </table>
 
-Table-Driven-Test makes it easy to add new test case.
+Table-Driven-Test makes it easy to add a new test case.
 
+We define the test case table as `map[string]func(*testing.T)test`, which is referred to the test case name and the test case implementation `tt`.
 
-We define the test case table as `map[string]func(*testing.T)test`, which is referred as the test case name and the test case implementation `tt`. 
 ```go
 tests := map[string]func(t *testing.T) test {
```

Comment:
Could you replate this one to the test template style ?
template style is this.
https://github.com/vdaas/vald/issues/325#issue-597153386

What do you think??

Reply:
Sorry I forgot to merge the changes from your template, I will merge it later